gpu/people/src/cuda/nvidia/NPP_staging.cu

```diff
@@ -38,7 +38,6 @@
  * Ported to PCL by Koen Buys : Attention Work in progress!
  */
 
-#include "NPP_staging.hpp"
 #include <pcl/gpu/people/src/cuda/nvidia/NPP_staging.hpp>
```

Comment:
If we're including a file from src, I think we need input from @haritha-j and @SergioRAgostinho

Reply:
This fails on the Ubuntu 18.04 Clang build:
```
/__w/1/s/gpu/people/src/cuda/nvidia/NPP_staging.cu:41:10: fatal error: 'pcl/gpu/people/src/cuda/nvidia/NPP_staging.hpp' file not found
#include <pcl/gpu/people/src/cuda/nvidia/NPP_staging.hpp>
```

Reply:
Yes, it'll because we haven't told CMake that header files can exist in that dir. We shouldn't use "filename" style includes because it'll cause issues for downstream users.

Hence labelled this line as needs feedback from people working actively on gpu module

Reply:
We aren't actively touching this part of the code so not much knowledge to pour in here. It looks like a private header as you pointed out.

I would not touch this in this PR. It will require changes of a different type that should likely not be buried in here.

Reply:
@gnawme Please undo any changes to the include, create an issue for tracking this and then resolve the conversation. Sorry for the confusion created by me

Reply:
Reverted, and opened #4310